test/specs/Loader.spec.js

```diff
@@ -36,6 +37,7 @@ beforeEach(function() {
       ],
       defaultTopicId: 'sometopic',
       translationFallbackCode: 'somelang',
+      languages: [navLang, 'rm', 'somelang'],
```

Comment:
I don't think we need `navLang` in this list because it's not true all the time. Put 'de','en' and 'fr' instead.